swarm/vulcan.1650532904.yml

```diff
@@ -19,10 +19,10 @@ services:
       - source: db-pw
         target: password
 
-  # Exists purely to allow updates to archimedes image on all possible nodes via service deployer.
+  # Exists purely to allow updates to archimedes and archimedes-node images on all possible nodes via service deployer.
```

Comment:
Okay, so I'm probably doing this wrong, but wanted to make sure that Vulcan pulls in `archimedes-node` in the swarm ... not sure how to resolve the conflict or what's the right way to make sure production gets the right images, @corps ?

Reply:
Yes, that is correct.  You can test this before merge by doing a `PUSH_IMAGES=1 IMAGES_PREFIX=etnaagent/ IMAGES_POSTFIX=:production make -C archimedes-node release` before hand (pushing the image to dockerhub) and then making this change inside of portainer under the vulcan stack.